### ISSUE TITLE
Set zoom limit based on layer tile

### ISSUE THREAD

**Issue by dhuangg** [ScandinavianSection-UCLA/Nexus2#30]
Right now you can zoom in/out as much as you want. But some maps only display limited area (i.e. Prussian map layer) or the map can't render a higher zoom level. I think would be helpful to have zoom limits based on the map layer's capability.

**Comment by tango2** [comment]
Yeah--there's kind of a weird mismatch too with those Prussian maps--they really only cover the contested part of Denmark that the Germans took in the war(s) of 1864.

**Comment by dhuangg** [comment]
@NeilAllavarpu perhaps you can do a write-up updating the challenges for this ticket
